multimodal/src/autogluon/multimodal/optimization/lit_distiller.py

```diff
@@ -246,6 +256,29 @@ def _compute_rkd_loss(
 
         return loss
 
+    def _compute_softmax_regression_loss(
+        self,
+        student_output: dict,
+        teacher_output: dict,
+    ):
+        student_feature = student_output[self.student_model.prefix][FEATURES].squeeze(dim=1)
+        student_feature = self.output_feature_adaptor(student_feature)
+
+        student_logits = self.teacher_model.head(student_feature)
+        soft_labels = teacher_output[self.teacher_model.prefix][LOGITS].squeeze(dim=1)
+
+        student_logits = student_logits / self.temperature
+        soft_labels = soft_labels / self.temperature
```

Comment:
The paper doesn't use temperature for mse.

Reply:
I think it's similar to soft logit loss so added a temperature here. I just removed it and will report the performance change in readme later.